### ISSUE TITLE
daily updates on dockerhub

### ISSUE THREAD

**Issue by lingej** [netbox-community/netbox-docker#954]
### Desired Behavior

Hello Netbox Team,
I have noticed that there is a daily update on dockerhub although no changes have been made in the repository.

I use newreleases.io to inform me about changes in the container. Since a few weeks/month I get daily information about new container versions.

Greetings
Jörg

### Contrast to Current Behavior

Shouldn't new containers only be created if something has really changed in the code?

**Comment by cimnine** [comment]
We had such checks, maybe they are broken.

Another reason could be that the base-image changed. This would trigger a build with updated dependencies, even if nothing changed in NetBox Docker or NetBox.

**Comment by tobiasge** [comment]
Something in the Docker registry API changed and the checks don't work anymore. See from the build log:
```
 Checking labels for 'netboxcommunity' and 'netbox'
  jq: error (at <stdin>:1): Cannot iterate over null (null)
  ./build-functions/get-public-image-config.sh: line 36: layers: bad array subscript
  jq: error (at <stdin>:1): Cannot iterate over null (null)
```

I think I will change the `curl` calls to `skopeo`.

**Comment by tobiasge** [comment]
Closed with #956 and #958
